style.css

```diff
@@ -590,26 +834,496 @@ a {
+    width: 75%;
+  }
+
+  /* Contact section */
+  .contact-section {
+    width: 85%;
   }
 }
 
-@media (max-height: 667px) {
+/* Desktop devices version */
+@media only screen and (min-width: 992px) {
+  /* Laptops/desktops */
+  :root {
+    --fs-text: 1.125em;
+    --fs-title: 2.75em;
+    --fw-bold: 600;
+  }
+
+  /* General Styles */
+  body {
+    font-family: var(--ff-text);
+    font-size: var(--fs-text);
+    font-weight: var(--fw-normal);
+  }
+
+  .button {
+    width: 7.5em;
+    height: 3.25em;
+    font-size: 0.8em;
+  }
+
+  .border {
+    padding-bottom: 4em;
+  }
+
+  /* Background glowing objects */
+  .circle-one {
+    top: 50px;
+    left: 80px;
+    width: 100px;
+    height: 100px;
+  }
+
+  .circle-two {
+    top: 500px;
+    right: 65px;
+    width: 120px;
+    height: 120px;
+  }
+
+  .circle-three {
+    width: 57.53px;
+    height: 700.92px;
+    left: -112px;
+    top: 700.12px;
+  }
+
+  .circle-four {
+    width: 108px;
+    height: 402px;
+    left: 1320.28px;
+    top: 1200.74px;
+    -webkit-transform: matrix(-0.84, -0.54, -0.54, 0.84, 0, 0);
+    -ms-transform: matrix(-0.84, -0.54, -0.54, 0.84, 0, 0);
+    transform: matrix(-0.84, -0.54, -0.54, 0.84, 0, 0);
+  }
+
+  .circle-five {
+    position: absolute;
+    width: 308px;
+    height: 155px;
+    left: -41px;
+    top: 2300px;
+  }
+
   .circle-six {
-    top: 3500px;
+    position: absolute;
+    width: 246.19px;
+    height: 916.38px;
+    left: 1090.3px;
+    top: 2484px;
+    -webkit-transform: matrix(-0.26, 0.97, 0.97, 0.26, 0, 0);
+    -ms-transform: matrix(-0.26, 0.97, 0.97, 0.26, 0, 0);
+    transform: matrix(-0.26, 0.97, 0.97, 0.26, 0, 0);
+  }
+
+  /* Header section */
+  .header-section {
+    font-size: var(--fs-text);
+    padding: 1.75em 6em;
+  }
+
+  .nav-list {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    list-style: none;
+  }
+
+  .nav-list-item {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+  }
+
+  .nav-list-item > a {
+    text-decoration: none;
+    color: var(--subtitle-color);
+  }
+
+  .nav-list-item + .nav-list-item {
+    margin-left: 1em;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  /* Main page content */
+  .main-section {
+    padding: 0 5em;
+  }
+
+  /* Headline section */
+  .headline-section {
+    width: 45vw;
+    min-width: 40vw;
+    max-width: 50vw;
+  }
+
+  #headline-title {
+    font-size: var(--fs-title);
+    letter-spacing: 3px;
+    line-height: 1.125;
+    font-weight: 500;
+    margin-top: 1em;
+  }
+
+  #headline-sub {
+    font-size: 1.85em;
+    margin-top: 0.25em;
+  }
+
+  .intro {
+    font-size: var(--fs-text);
+    text-align: justify;
+    line-height: 1.5;
+    margin-top: 2.5em;
+    padding: 0 0.75em;
+  }
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, you are to stick to the design as much as possible. Please align the text center to match the original design.😊

Reply:
Actually, the text-align property changes depending on the device the user is on.
That's why in the desktop version media query it is set to justify. 😕 

### Here are the links to the project's requirements and the Figma design:
> Kindly make sure you check template 4 🥰 
- [The requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)
- [Template 4](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)